### DISCUSSION TITLE
Swagger endpoint

### DISCUSSION THREAD

**Discussion by cannahum** [oapi-codegen/oapi-codegen#1106] [Q&A] [answered]
After implementing all of the generated interfaces and starting the server, is there a Swagger endpoint that we can reach? I'm using chi and I haven't found anything in documentation, discussions or pull requests about this.

**Comment by jamietanna** [comment, marked as answer]
Hey, see https://github.com/deepmap/oapi-codegen/issues/746#issuecomment-1241140748 - this is something that's not expected to be part of this project, but can be added on, as mentioned in the linked comments 😄
